Sources/CoffeeBot/CoffeeBot.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  CoffeeBot.swift
+//  CoffeeBot
+//
+
+import LoggerAPI
+import BotsKit
+
+internal protocol CoffeeBotInputProtocol {
+    func send(activity:Activity)
+}
+
+public final class CoffeeBot : Bot {
+    
+    //MARK: Bot Protocol Properties
+    public let name = "CoffeeBot"
+    public var sendActivity: Signal<Activity> = Signal()
+    
+    //MARK: Other Properties
+    private let coffeeManager : CoffeeManager
+    
+    //MARK: Bot Protocol Methods
+    public func dispatch(activity: Activity) -> DispatchResult {
+        self.coffeeManager.process(activity: activity)
+        return .ok
+    }
+    
+    //MARK: Lifecycle
+    public init() {
```

Comment:
I think you are right I will create a factory to create CoffeeBot, in this way we will be able to inject dependencies.
I would like to create own factory class because I think external classes don't necessarily know about all internal types such as EntityRecognitionService.